lms/djangoapps/teams/static/teams/js/spec/views/team_profile_spec.js

```diff
@@ -62,16 +66,55 @@ define([
                     view = createTeamProfileView(requests, {});
                 expect(view.$('.discussion-thread').length).toEqual(3);
             });
+
+            it('shows New Post button when user joins a team', function () {
+                var requests = AjaxHelpers.requests(this),
+                    view = createTeamProfileView(requests, {});
+                var membership = [
+                    {
+                    'user': {
+                        'username': 'bilbo',
+                        'profile_image': {
+                            'has_image': true,
+                            'image_url_medium': '/image-url'
+                        }}
+                    }
+                ];
+
+                expect(view.$('.new-post-btn').length).toEqual(0);
+                teamModel.set('membership', membership);
+                expect(view.$('.new-post-btn').length).toEqual(1);
+            });
+
+            it('hides New Post button when user left a team', function () {
+                var requests = AjaxHelpers.requests(this),
+                    membership = [
+                        {
+                        'user': {
+                            'username': 'bilbo',
+                            'profile_image': {
+                                'has_image': true,
+                                'image_url_medium': '/image-url'
+                            }}
```

Comment:
This dict is being used in multiple places in this file. Can you make this a constant?